lib/core/providers/cards.dart

```diff
@@ -22,13 +22,11 @@ class CardsDataProvider extends ChangeNotifier {
   }
 
   /// DEFAULT STATES
-  bool _noInternet = false;
-  bool _isLoading = false;
-  Map<String, bool> _cardStates = {};
-  Map<String, CardsModel?> _webCards = {};
-
+  bool? _isLoading = false;
```

Comment:
This change makes _isLoading nullable, which is generally a regression unless absolutely required.